pants

```diff
@@ -51,17 +55,24 @@ function get_exe_path_or_die {
   fi
 }
 
+if [[ -f "${PANTS_INI}" ]]; then
+  pants_config_file="${PANTS_INI}"
+fi
+if [[ -f "${PANTS_TOML}" ]]; then
+  pants_config_file="${PANTS_TOML}"
+fi
+
 function get_pants_config_value {
   config_key="$1"
   optional_space="[[:space:]]*"
-  if [[ -f 'pants.ini' ]]; then
+  if [[ "${pants_config_file}" == *.ini ]]; then
```

Comment:
I think that this will fail to match if there are multiple ini files? It seems like continuing to match using `-f` makes sense here?

Reply:
This is a string match. We already checked that this file exists above. This just checks if it has a .ini suffix or a .toml suffix, which is all we care about here.

Reply:
Got it.

Reply:
Whether or not there are multiple ini files on disk is not known to this script (wasn't before, and isn't now). We just pick one that we grep the version out of, and this change allows that one to be configurable.